packages/caliper-ethereum/test/utils/invalidUrlConfig.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Could we move the network configuration files to a subdirectory called `sample-configs` as that is similar to the fabric tests. These files are not util files. We want to avoid the general term utils if we can as it's also not very descriptive.

Reply:
Yeah Sure. I'll cover it now!!